src/version.h

```diff
@@ -4,15 +4,58 @@
 #ifndef BITCOIN_VERSION_H
 #define BITCOIN_VERSION_H
 
-#include "clientversion.h"
+#if defined(HAVE_CONFIG_H)
+#include "config/gridcoin-config.h"
+#else
+
+//! Copyright string used in Windows .rc files
+#define COPYRIGHT_STR "2009-"
+
+//These need to be macros, as version.cpp's and bitcoin-qt.rc's voodoo requires it
+#define CLIENT_VERSION_MAJOR       3
+#define CLIENT_VERSION_MINOR       6
+#define CLIENT_VERSION_REVISION    3
+#define CLIENT_VERSION_BUILD       17
```

Comment:
Having the version in two places is going to cause problems. Remove it from here, or from the .am file that was merged lately.

Reply:
Removing this will break compatibility with the qmake build. Is this fine @denravonska ?